ghcide/ghcide.cabal

```diff
@@ -18,11 +18,6 @@ tested-with:        GHC ==9.10.1 || ==9.8.2 || ==9.6.5 || ==9.4.8
 extra-source-files:
   CHANGELOG.md
   README.md
-  test/data/**/*.cabal
-  test/data/**/*.hs
-  test/data/**/*.hs-boot
-  test/data/**/*.project
-  test/data/**/*.yaml
```

Comment:
These must be added to `haskell-language-server.cabal`, otherwise we can't run the tests from a `cabal sdist` tarball.

Reply:
Yes, done